python-sdk/example_dags/example_amazon_s3_snowflake_transform.py

```diff
@@ -19,7 +20,10 @@ def combine_data(center_1: Table, center_2: Table):
     UNION SELECT * FROM {{center_2}}"""
 
 
-@aql.transform(assume_schema_exists=True)
+@aql.transform(
+    assume_schema_exists=True,
+    query_modifier=QueryModifier(pre_queries=["ALTER SESSION SET query_tag='not_guinea_pig';"]),
```

Comment:
@tatiana We ran this DAG on Astro but like are still getting this error 
```
Actual statement count 2 did not match the desired statement count 1.
```

Reply:
@pankajastro, as we spoke in the call, the issue seems that the Astro CI is pointing to an old version of the Python SDK. The stack trace had:
```
  File "/home/astro/.local/lib/python3.10/site-packages/astro/databases/base.py", line 141, in run_sql
    result = self.connection.execute(
```

Which does not match this version (or the main branch):
https://github.com/astronomer/astro-sdk/blob/main/python-sdk/src/astro/databases/base.py#L106-L128

Please, let me know if that's not the issue and we can investigate further#1962